We do not have packaged furo and it is not needed to run tests.

--- zope_proxy-7.2/pyproject.toml.orig
+++ zope_proxy-7.2/pyproject.toml
@@ -43,7 +43,6 @@
 docs = [
   "Sphinx",
   "repoze.sphinx.autointerface",
-  "furo",
 ]
 # We have a circular dependency with zope.security for testing
 test = [
